admin/routes/school.js

```diff
@@ -41,6 +42,7 @@ const school = (router) => {
   router.post('/group-pupils/add', isAuthenticated(rolesConfig.ROLE_TEACHER), (req, res, next) => group.addGroup(req, res, next))
   router.post('/group-pupils/edit', isAuthenticated(rolesConfig.ROLE_TEACHER), (req, res, next) => group.editGroup(req, res, next))
   router.get('/group-pupils/delete/:groupId', isAuthenticated(rolesConfig.ROLE_TEACHER), (req, res, next) => group.removeGroup(req, res, next))
+  router.get('/download-guidance', isAuthenticated(rolesConfig.ROLE_TEACHER), (req, res, next) => downloadFile(req, res, next))
```

Comment:
shouldn't that be `downloadGuidance`?
`downloadFile` sounds too generic to me

Reply:
@GeorgeChatzigiannis I'm expecting to extend this controller after the feb trial with some other downloads that need refactoring, that's why the generic name.

Reply:
@pris54 why not creating a controller called related (or something more distinctive if you wish) and include a method which is very specific?